plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineDeployOutput.java

```diff
@@ -30,12 +30,12 @@
   private static class Version {
     String id;
     String service;
+    String project;
   }
 
   private List<Version> versions;
 
   private AppEngineDeployOutput() {
```

Comment:
to prevent direct instantiation and require clients to go through the factory method